src/mobile-ui-react/ResizablePanel.tsx

```diff
@@ -129,14 +131,17 @@ export function ResizablePanel(props: ResizablePanelProps) {
         updateMaxHeight(currHeight);
       }
     }, 0);
-  }, [divRef, minInitialHeight, maxInitialHeight, initialMaxHeightSet, updateMaxHeight, props.heightCanExceedContents, setHeightAndCallOnResized]);
+  }, [divRef, minInitialHeight, maxInitialHeight, initialMaxHeightSet, updateMaxHeight, props.heightCanExceedContents, setHeightAndCallOnResized, isMountedRef]);
 
   const onWindowResize = React.useCallback(() => {
-    const safeAreaOffsets = getCssVariableAsNumber("--itm-safe-area-top") + getCssVariableAsNumber("--itm-safe-area-bottom");
-    const newMaxHeight = window.outerHeight - 100 - safeAreaOffsets;
-    if (maxHeight !== undefined && maxHeight > newMaxHeight)
-      setMaxHeight(newMaxHeight);
-  }, [maxHeight]);
+    setTimeout(() => {
```

Comment:
I see you added code that will only run when the component is mounted, and you also delay the code through setTimeout. Are both these needed to fix the problem? The former makes sense, but I don't understand why the setTimeout wrappers are needed.

Reply:
The setTimeout is the change that fixed the problem. As far as I can tell, the problem was that the processing of the onWindowResize would sometimes happen too early, and delaying it fixed this. (Note that the delay is 0, so it only pushes the call back until the current processing is done.) To be honest, I don't know why the problem occurred in the first place, but this delay fixed it for me.

The isMountedRef check inside this setTimeout callback was added because that should always be done. The isMountedRef checks inside the other setTimeout callbacks were added because they always should have been there, but were missing. (In other words, they were a bug fix for a completely unrelated bug that we just hadn't noticed before now.)